src/content/concepts/hot-module-replacement.md

```diff
@@ -64,7 +64,7 @@ A `check` makes an HTTP request to the update manifest. If this request fails, t
 
 The `apply` method flags all updated modules as invalid. For each invalid module, there needs to be an update handler in the module or in its parent(s). Otherwise, the invalid flag bubbles up and invalidates parent(s) as well. Each bubble continues until the app's entry point or a module with an update handler is reached (whichever comes first). If it bubbles up from an entry point, the process fails.
 
-Afterwards, all invalid modules are disposed (via the dispose handler) and unloaded. The current hash is then updated and all `accept` handlers are called. The runtime switches back to the `idle` state and everything continues as normal.
+Afterward, all invalid modules are disposed (via the dispose handler) and unloaded. The current hash is then updated and all `accept` handlers are called. The runtime switches back to the `idle` state and everything continue as normal.
```

Comment:
Afterwards was just fine imo, requesting help from another grammar expert 😓 Also continues sound ok

Reply:
@EugeneHlushko According to me, `modules` being a cluster or plural, `afterwards` with it isn't grammatically correct, same for `continues`. It is subject-verb agreement. Though many a time, afterwards is considered British, not American.

Reply:
@wizardofhogwarts `afterwards` makes more sense in this context IMO.
```suggestion
Afterwards, all invalid modules are disposed (via the dispose handler) and unloaded. The current hash is then updated and all `accept` handlers are called. The runtime switches back to the `idle` state and everything continue as normal.
```
cc @EugeneHlushko